### PR TITLE
Update document v0.4

### PR DESCRIPTION
**Pull request by SparkSnail**
1. move nnictl folder
2. delete kubernetsServer in nnictl
3. refactor aks document
4. add warning information to expand relative path
5. update experiment status when the experiment crashed.

### CODE REVIEW

File: docs/KubeflowMode.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 ===
 Now NNI supports running experiment on [Kubeflow](https://github.com/kubeflow/kubeflow), called kubeflow mode. Before starting to use NNI kubeflow mode, you should have a kubernetes cluster, either on-prem or [Azure Kubernetes Service(AKS)](https://azure.microsoft.com/en-us/services/kubernetes-service/), a Ubuntu machine on which [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) is installed and configured to connect to your kubernetes cluster. If you are not familiar with kubernetes, [here](https://kubernetes.io/docs/tutorials/kubernetes-basics/) is a goot start. In kubeflow mode, your trial program will run as kubeflow job in kubernetes cluster. 
 
-## Prerequisite
+## Prerequisite for on-premises
```
Comment (yds05) [inline docs/KubeflowMode.md]: How about 'Prerequisite for on-premises Kubernetes Service'?

Comment (SparkSnail) [reply docs/KubeflowMode.md]: Updated to  'Prerequisite for on-premises Kubernetes Service'

File: docs/ExperimentConfig.md [hunk 2]
```diff
@@ -393,6 +389,18 @@ machineList:
 
       __name__ is the value of ```--name``` used in az command.
 
+  * __azureStorage__
+    
+    If users use azure kubernets service, they should set azure storage account to store code files.
```
Comment (chicm-ms) [inline docs/ExperimentConfig.md]: typo: kubernets

Comment (SparkSnail) [reply docs/ExperimentConfig.md]: updated to kubernetes

**Pull request merged**